Bash/README.md

```diff
@@ -49,8 +49,13 @@ cleaning up...
 20221103-085511 -- =========== INSTALL FINISHED AT 20221103-085511 ===============
 ```
 
-## Huntress PPPC
-The `Huntress PPPC.mobileconfig` is designed to upload/import to your MDM and can then be scoped to a collection of devices. This payload will grant the Huntress agent FDA (full disk access) and allow for the optimal performance of the Huntress agent.
+## Huntress Mobileconfigs
 
-Further details: [Generic Deployment and PPPC Payload for Full Disk Access](https://support.huntress.io/hc/en-us/articles/10962515436691)
+The `mobileconfig` files in this directory are preconfigured settings for MDMs.
+They are designed to upload/import to your MDM and can then be scoped to a collection of devices.
 
+- `HuntressSystemExtension.mobileconfig`: This payload grants all the permissions required by the Huntress Agent and the Huntress System Extension. It enables FDA (full disk access) for both and provides the System Extension with the neccessary permissions for install and operation. Most users will choose this configuration.
+
+- `Huntress PPPC.mobileconfig`: This payload will grant the Huntress agent FDA (full disk access). **This file does not configure the System Extension** and should be used for deployments that only use the Huntress Agent. Use `HuntressSystemExtension.mobileconfig` instead if you want the full capabilities of Huntress.
+
+- `HuntressLegacyAgentProfile.mobileconfig`: A legacy profile used for versions of the Huntress macOS Agent before version `0.13.72`. Kept around for historical reasons, no customers should use this unless expressly directed to by Huntress Support.
```

Comment:
Should these two be listed separately (in, say, a "legacy configs" or something) so people don't assume they should install all three?

Reply:
Good idea